evalai/challenges.py

```diff
@@ -11,66 +11,43 @@
 
 @click.group(invoke_without_command=True)
 @click.pass_context
-def challenges(ctx):
-    """
-    Challenges and related options.
-    """
-    if ctx.invoked_subcommand is None:
-        welcome_text = ("Welcome to the EvalAI CLI. Use evalai"
-                        "challenges --help for viewing all the options.")
-        echo(welcome_text)
-
-
-@click.group(invoke_without_command=True, name='list')
-@click.pass_context
 @click.option('--participant', is_flag=True,
               help="Show the challenges that you've participated")
 @click.option('--host', is_flag=True,
               help="Show the challenges that you've hosted")
-def list_challenges(ctx, participant, host):
+def challenges(ctx, participant, host):
     """
     Used to list challenges.
-    Invoked by running `evalai challenges list`
+    Invoked by running `evalai challenges`
     """
-    if participant:
-        get_challenge_count(host, participant)
-    elif host:
+    if participant or host:
         get_challenge_count(host, participant)
     elif ctx.invoked_subcommand is None:
         get_challenge_list()
 
 
-@click.command(name='ongoing')
-def list_ongoing_challenges():
+@challenges.command()
+def ongoing():
     """
     Used to list all the challenges which are active.
```

Comment:
<img width="720" alt="screen shot 2018-06-10 at 4 13 18 pm" src="https://user-images.githubusercontent.com/12206047/41200821-727ee496-6cc9-11e8-891e-67b516fc4290.png">
I think either we need to shorten the text or increase the word limit.